src/index.html

```diff
@@ -10,23 +10,28 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Two Search Bars</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-conteiner">
```

Comment:
There is a typo in the class name 'search-conteiner'. It should be 'search-container'.